action.yml

```diff
@@ -9,6 +9,10 @@ inputs:
     description: 'Documentation id. Can be found in the documentation settings on https://bump.sh'
   token:
     description: 'Documentation token. Can be found in the documentation settings on https://bump.sh'
+  hub:
+    description: 'Hub slug or id. Needed when deploying to a documentation attached to a Hub. Can be found in the hub settings on https://bump.sh'
+  branch:
+    description: 'Branch name used during `deploy` or `diff` commands. This can be useful to maintain multiple API reference history and make it available in your API documentation.'
```

Comment:
Maybe we could mention here that default branch is used when this parameter is not provided ?

Reply:
I didn't find a good wording so I've merged with the wording from the README. Maybe we can adapt it later if you think necessary?